### DISCUSSION TITLE
WASM: Where does the wwwroot/ hide?

### DISCUSSION THREAD

**Discussion by pr8x** [AvaloniaUI/Avalonia#11966] [General]
I am a complete noob when it comes to WebAssembly, so bear with me 😅

I used the new Avalonia Template Studio to create a Browser/Wasm Avalonia project. That compiles and runs fine - even with AOT. Now I am trying to serve some static files (so I can HTTP/GET them from my WASM app) by putting the inside the wwwroot folder. But I can't seem to find it? Did a quick search and it seems that the Avalonia.Browser nuget package has one, but I am not sure if it's the right one.

**Comment by Gillibald** [comment]
I think you have to explicitly declare static content like so: https://github.com/AvaloniaUI/Avalonia/blob/master/src/Browser/Avalonia.Browser/Avalonia.Browser.csproj#L37

The wwwroot folder is just some naming convention

**Reply by pr8x** [reply]
It seems that anything directly inside the Project/AppBundle/ folder will actually be hosted. However subfolders don't seem to work here. `WasmExtraFilesToDeploy`  is the  key here I guess, but there is exactly ZERO documentation on it... Well.

**Reply by maxkatz6** [reply]
Microsoft will likely add support for classic Content build-action on the wasm as well. Probably that's why they didn't document WasmExtraFilesToDeploy yet.
But anyway, with WasmExtraFilesToDeploy you can configure folder with TargetPath property
something like:
```
<WasmExtraFilesToDeploy Include="..\..\assets\Definitions\*.zip" TargetPath="Definitions\%(Filename)%(Extension)" />
```
